### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by darkowlzz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
 ## Overview
 
 The purpose of liveness probe is to be able to detect liveness of CSI driver
-and in case of as a driver's failure report it by returning non zero return code.
+and in case of a driver's failure, report it by returning non zero return code.
 Livenessprobe leverages CSI Probe API call, which must be answered by CSI
 compatible driver.
-Liveness probe is meant to be used in pair with kubelet's LinessProbe hook, which
-executes it periodically and check the rerutn code. Non zero return code indicates
+Liveness probe is meant to be used in pair with kubelet's LivessProbe hook, which
```
Comment (sbezverk) [inline README.md]: You have type here too ;)

Comment (darkowlzz) [reply README.md]: oops! :D updated

**Comment by jsafrane** [comment]
@sbezverk, you must use `/approve`, I am not powerful enough.

**Pull request merged**